filter/kubernetes_filter.go

```diff
@@ -24,7 +24,8 @@ import (
 )
 
 const (
-	WsIdLabel         = "che.workspace_id"
+	DefaultWsIdLabel  = "che.workspace_id"
+	WsIdEnvVar        = "CHE_WORKSPACE_ID_LABEL"
```

Comment:
What do you think if we use some common naming for that configuration, like `TARGET_POD_SELECTOR`...

Later we can do a similar object selector for filtering containers... Currently machine-exec without containers specified resolves only containers with `CHE_MACHINE_NAME` env var

Reply:
I'm not entirely sure what you mean, do you mean having TARGET_POD_SELECTOR become an env var and then if TARGET_POD_SELECTOR is set we filter off of that and if we don't we filter off of `che.workspace_id:$(CHE_WORKSPACE_ID)"? I think that could work